backend-implicaction/src/main/java/com/dynonuggets/refonteimplicaction/service/PostService.java

```diff
@@ -64,4 +68,19 @@ public PostResponse getPost(Long postId) {
     private PostResponse getPostResponse(Post post) {
         return postAdapter.toPostResponse(post, commentService.commentCount(post), voteService.isPostUpVoted(post), voteService.isPostDownVoted(post));
     }
+
+    public List<PostResponse> getLastPosts(int postCount) {
+        List<Post> lastPosts = postRepository.findAll(Sort.by(Sort.Direction.DESC, "createdAt"));
+        if (lastPosts.size() < postCount) {
+            postCount = lastPosts.size();
+        }
+        lastPosts = lastPosts.subList(0, postCount);
+
+        final List<PostResponse> returnList = new ArrayList<>();
+
+        for (Post post : lastPosts) {
+            returnList.add(postAdapter.toPostResponse(post, 0, false, false));
+        }
+        return returnList;
```

Comment:
`return postRepo.findAll(Pageable.of(0, postCount, Sort.by(Sort.Direction.Desc, "createdAt")).map(....)`